src/ut/call_list_store_test.cpp

```diff
@@ -43,36 +43,60 @@
 #include "mock_cassandra_store.h"
 #include "cass_test_utils.h"
 #include "mock_sas.h"
+#include "mock_cassandra_connection_pool.h"
+#include "mock_a_record_resolver.h"
+#include "fake_base_addr_iterator.h"
 
 #include "call_list_store.h"
 #include "mementosasevent.h"
 
+
 using namespace CassTestUtils;
 
 const SAS::TrailId FAKE_TRAIL = 0x123456;
 
 // The class under test.
-//
-// We can't test the call list store directly as we need to supply a mock
-// client, and therefore override the get_client() and relese_client()
-// methods.
+// We don't test the Cache class directly as we need to use a
+// MockCassandraConnectionPool that we can use to return MockCassandraClients.
+// However all other methods are the real ones from Cache.
 class TestCallListStore : public CallListStore::Store
 {
 public:
-  MOCK_METHOD0(get_client, CassandraStore::Client*());
-  MOCK_METHOD0(release_client, void());
+  void set_conn_pool(CassandraStore::CassandraConnectionPool* pool)
```

Comment:
You have this method in several places - is it worth having a common shared super class in cpp-common?